src/scripts/run-tests.sh

```diff
@@ -60,7 +60,33 @@ if [[ "$os" = CYGWIN* || "$os" = MINGW* || "$os" = MSYS* ]]; then
     binext=".exe"
 fi
 
-"${tmpdir}/bin/run-matlab-command$binext" "\
+if [[ -n "${PARAM_SPLIT_TYPE}" ]]; then
```

Comment:
I feel we should avoid string literals being repeated. Instead we can break the complete string into substrings and add if-else between them as required